chimerapulse/core/language/sentiment_analysis.py

```diff
@@ -0,0 +1,52 @@
+"""Module that calls Azure Sentiment Analysis
+https://learn.microsoft.com/en-us/azure/ai-services/language-service/sentiment-opinion-mining/overview?tabs=prebuilt
+
+Example:
+    python -m chimerapulse.core.language analyzesentiment -t 'The food was great'
+"""
+
+import os
+
+import click
+from dotenv import load_dotenv
+
+from azure.ai.textanalytics import TextAnalyticsClient
+from azure.core.credentials import AzureKeyCredential
+
+# Load env vars
+load_dotenv()
+
+@click.command()
+@click.option('-t', 'text', required=True, help='Text to analyze')
+def analyzesentiment(text):
+    language_analyzesentiment(text)
+    
+def __authenticate_client():
+    """Authenticate service
+
+    Returns:
+        (TextTranslationClient): Authenticated client
+    """
+    # Get Configuration Settings
+    languageendpoint = os.getenv('TRANSLATOR_SERVICE_ENDPOINT')
+    languagekey = os.getenv('TRANSLATOR_SERVICE_KEY')
+
+    credential = AzureKeyCredential(languagekey)
+    text_analytics_client = TextAnalyticsClient(
+            endpoint=languageendpoint, 
+            credential=credential)
+    return text_analytics_client
+
+def language_analyzesentiment(text):
```

Comment:
this should be fine for now but will need to modify to return sentiment in a JSON order for chained capabilities to process